spec/components/dropdown/Dropdown.spec.tsx

```diff
@@ -175,18 +176,39 @@ describe('Dropdown component test suite =>', () => {
       userEvent.type(input,'B');
       expect(queryByText(/Search for term/)).toBeInTheDocument();
     });
-    it('should focus on the first option', async () => {
-      const {  getByText } = render(
-        <Dropdown
-          options={dropdownProps.options}
-          displayArrowIndicator
-          enableTermSearch
-        />
-      );
-      const input = screen.getByRole('textbox');
-      userEvent.type(input,'B');
-      expect(getByText('banana').className).toContain('tk-select__option--is-focused');
+ 
+
+    describe('when focusing on the dropdown list', () => {
+      it('should focus on the first option (skiping the headerOption) when there is a value on the input', async () => {
+        const {  getByText } = render(
+          <Dropdown
+            options={dropdownProps.options}
+            displayArrowIndicator
+            enableTermSearch
+          />
+        );
+        const input = screen.getByRole('textbox');
+        userEvent.type(input,'B');
+        expect(getByText('banana').className).toContain('tk-select__option--is-focused');
```

Comment:
to not rely on the class, you can also compare your element to document.active

Reply:
The active element is the input from the dropdown.

Reply:
ooooh! make sense